ga4gh/server/datamodel/peers.py

```diff
@@ -0,0 +1,104 @@
+"""
+Peer datamodel for exchanging data about GA4GH services.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+import re
+
+import ga4gh.server.exceptions as exceptions
+
+import ga4gh.schemas.protocol as protocol
+
+
+def isUrl(urlString):
```

Comment:
Yeah, instead of a regex you can just catch an exception from `urllib`, I'll do that.

http://grokbase.com/t/python/python-list/018fex83ty/check-url-simply

```
import urllib

def checkURL(url):
try: urllib.urlopen(url)
except IOError: return 0
return 1

if __name__ == '__main__':
assert checkURL('http://slashdot.org')
assert not checkURL('http://foo')

HTH.
Alex.
```